### ISSUE TITLE
Forms Dashboard Update - [Debt Resolution -FSR (Financial Status Report)]

### ISSUE THREAD

**Issue by Tom-Davis** [department-of-veterans-affairs/va.gov-team#59520]
# Google Analytics Implementation or QA Request

## What this form is for

<details>
<summary>More Information</summary>
<div>
    <br>
<p>Use this template to request Google Analytics implementation or QA for your product. Please fill out all brackets.</p>
    
> _Please see the following information for implementation timeline:_
> New Teams: Implementation will usually be completed in 4 sprints
> 
> - Sprint 1: Discovery/Orientation
> - Sprint 2: FE Implementation
> - Sprint 3: Tagging Implementation
> - Sprint 4: QA
> 
> All other teams: Implementation will usually be completed in 2 sprints
> 
> - Sprint 1: Discovery and FE Implementation
> - Sprint 2: Tagging Implementation and QA

</div>
</details>

### Required
- **Is this request for **Google Analytics implementation** or **QA**?**

     - [ ] Google Analytics implementation
     - [X] QA

- **Who is this request for?**

    - Debt Resolution
    
    
- **Provide the link(s) for Google Analytics implementation or QA**
  
    - Current URLs:
'/introduction'
'/veteran-information'
'/all-available-debts'
'/contact-information'
'/employment'
'/employment-records'
'/income/:index'
'/benefits'
'/social-security'
'/additional-income'
'/additional-income-records'
'/spouse-information'
'/spouse-employment-history'
'/spouse-employment'
'/spouse-employment-records'
'/spouse/income/:index'
'/spouse-benefits'
'/spouse-benefit-records'
'/spouse-social-security'
'/spouse-social-security-records'
'/spouse-additional-income'
'/spouse-additional-income-records'
'/dependents'
'/dependent-records'
'/monetary-assets'
'/real-estate-assets'
'/real-estate-asset-records'
'/vehicles'
'/vehicle-records'
'/recreational-vehicles'
'/cfsr-recreational-vehicle-records'
'/other-assets'
'/other-asset-records'
'/expenses-explainer'
'/expenses'
'/utilities'
'/utility-records'
'/repayments'
'/repayment-records'
'/other-expenses'
'/other-expense-records'
'/option-explainer'
'/resolution-option/:index'
'/resolution-comment/:index'
'/resolution-waiver-agreement/:index'
'/resolution-comments'
'/bankruptcy-history'
'/bankruptcy-history-records'
'/review-and-submit'
'/confirmation'

    - URLs after enhancements goes live (currently on staging)
'/introduction'
'/veteran-information'
'/all-available-debts'
'/current-contact-information'
'/edit-mobile-phone'
'/edit-email-address'
'/edit-mailing-address'
'/employment-question'
'/enhanced-employment-records'
'/gross-monthly-income'
'/deduction-checklist'
'/deduction-values'
'/employment-history'
'/your-benefits'
'/edit-benefits'
'/additional-income-checklist'
'/additional-income-values'
'/other-income-summary'
'/add-other-income'
'/spouse-information'
'/spouse-name'
'/enhanced-spouse-employment-question'
'/enhanced-spouse-employment-records'
'/spouse-gross-monthly-income'
'/spouse-deduction-checklist'
'/spouse-deduction-values'
'/spouse-benefits'
'/spouse-benefit-records'
'/spouse-additional-income-checklist'
'/spouse-additional-income-values'
'/spouse-other-income-summary'
'/spouse-add-other-income'
'/dependents-count'
'/dependent-ages'
'/monetary-asset-checklist'
'/monetary-asset-values'
'/enhanced-real-estate-assets'
'/enhanced-real-estate-asset-records'
'/vehicles'
'/your-vehicle-records'
'/vehicles-summary'
'/recreational-vehicles'
'/cfsr-recreational-vehicle-records'
'/other-assets-checklist'
'/other-assets-values'
'/other-assets-summary'
'/add-other-asset'
'/expenses-explainer'
'/household-expenses-checklist'
'/household-expenses-values'
'/utility-bill-checklist'
'/utility-bill-values'
'/utility-bill-summary'
'/add-utility-bill'
'/credit-card-bills'
'/your-credit-card-bills'
'/credit-card-bills-summary'
'/installment-contracts'
'/your-installment-contracts'
'/installment-contracts-summary'
'/other-expenses-checklist'
'/other-expenses-values'
'/other-expenses-summary'
'/add-other-expense'
'/option-explainer'
'/resolution-option/:index'
'/resolution-comment/:index'
'/resolution-waiver-agreement/:index'
'/resolution-comments'
'/bankruptcy-history'
'/enhanced-bankruptcy-history-records'
'/review-and-submit'
'/confirmation'
'/employment-work-dates'
'/spouse-employment-work-dates'
    
- **When is the launch date?**
    - [Fiscal year Q3 (the end of June)]

- **What interactions need to be tracked?** _NOTE: [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) are automatically tracked_
    
    - [If this is a funnel, insert each step's URL - pages visible depending on the applicant's answers should be marked accordingly]
    
    - [Insert other interactions - please review the [Design System Components](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Design-System-Components.1840808040.html) and [GA Data Dictionary](https://depo-platform-documentation.scrollhelp.site/analytics-monitoring/Google-Analytics-Data-Dictionary.1810464912.html)  to make sure these interactions aren't already tracked]

### Optional
- **If applicable, what test users are needed?**  _NOTE: Do not put staging credentials in this ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository_

    - [Insert test user]

- **Any additional comments**

    - The debt team is making some enhancements and we want to ensure that all current analytics are still functioning as intended on all the post enhancement pages.

> Please leave the following blank

## Acceptance criteria
- [ ] All appropriate issue tagging is completed
- [ ] QA has been completed
- [ ] Events have been added to `Product` and `Benefit Hub` Content Groups

## Additional information

Please refer to [Platform Collaboration Cycle](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/index.html) or the [Analytics Request Touchpoint](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Analytics-request.1782120453.html) on Platform website for more information about the Collaboration Cycle.

**Comment by HeatherRienks** [comment]
following :)

**Comment by michelle-dooley** [comment]
@Tom-Davis - Here is the [current list](https://docs.google.com/spreadsheets/d/1Udi5r5s8OFo7YQ1Y3mJf0AjOsxf25QrgXmFkY0tqceY/edit?usp=sharing) of urls and events that are part of the [Combined Debt Portal KPIs dashboard](https://va-gov.domo.com/page/1976847942).  Are there new ones that are not covered by these or are all the new ones listed above a substring of one of these?  Are there any net new urls or events that need to be added to this list?

**Comment by Dr-Pongo** [comment]
Hey @michelle-dooley! The URLs Tom listed in the ticket are tacked on to `va.gov/manage-va-debt/request-debt-help-form-5655/` (example: `va.gov/manage-va-debt/request-debt-help-form-5655/introduction`), so it's separate from the Combined Debt Portal URLs. These are for the Financial Status Report (form 5655).

**Comment by michelle-dooley** [comment]
If you can provide that updated list this week we can get it added to our sprint starting tomorrow.

**Comment by Dr-Pongo** [comment]
@michelle-dooley I'm reviewing this with @denisecoveyduc shortly and I'll get back to you asap!

**Comment by Dr-Pongo** [comment]
Hey @michelle-dooley ! So, while we're doing the percentage roll out so ideally we'd like to track two sets of URLs. Before I go adding a bunch to that spreadsheet I wanted to confirm it won't break anything in place. Should I just add the new URL's at the end of the existing ones, or does order matter here? 

If it helps @amponce put together a list of our URLs, and all of the new ones for this release [here](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/60338). **Note:** `old` includes the original launch URLs, `combined` is what's currently live, and `enhanced` is what we're adding with this release. 

Please let me know if I should just add the new URLs to the existing list, or if we need to do something special moving forward, and we can always do a quick 5 min chat if you have the time and it will be easier to navigate. Thank you in advance! 

CC: @denisecoveyduc @HeatherRienks @Tom-Davis

**Comment by michelle-dooley** [comment]
@Dr-Pongo - we will have version 1 and version 2 of the funnel.  So any history will not be lost with changes (v1) and new urls (v2) are included going forward.  To keep it clean could you add a new tab to that worksheet for v2 version.  Then we can easily track each.  Example of how we have set up for other forms... 
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/93597659/20ffdbf1-cd31-4470-9a64-d22258cdcc20)

**Comment by michelle-dooley** [comment]
https://vfs.atlassian.net/wiki/spaces/AT/pages/1510408350/How+to+-+Loading+a+new+funnel+to+the+Forms+KPI+Dashboard

**Comment by Dr-Pongo** [comment]
Hey @michelle-dooley I added a new sheet to that existing one! I cloned the original, but also included an optional column! I hope that's alright, I stole it from someone else

https://docs.google.com/spreadsheets/d/1ZlwjHdCAZZLp2k7QRX0Bf0xsEBr98W_UYUW_rjW7Hlk/edit#gid=1132090508

CC: @HeatherRienks @Tom-Davis @amponce @OptionSelect

**Comment by Dr-Pongo** [comment]
Hey @michelle-dooley ! I just wanted to follow up on this to see when the info would be available (and where to view it)

**Comment by dorseypa1** [comment]
@ChristineYoungZA , it looks like the link to the customer googlesheet for the form updates is attached a few comments above.  Let me know when you have the input spreadsheet ready for BQ and then I'll load it.

**Comment by denisecoveyduc** [comment]
Hey @ChristineYoungZA and @dorseypa1 Do you think you will be able to do that today? We are hoping to go live with the deploy this afternoon. I assume after you have done that then the V2 version will be seen on the Domo dashboard for Form 5655 and we will be able to track the new experience, is that correct? 
cc: @michelle-dooley, @Dr-Pongo , @HeatherRienks @Tom-Davis

**Comment by dorseypa1** [comment]
Once the new version is loaded into BigQuery we will refresh the Domo dataset and all of the new experience should be there to track.

**Comment by ChristineYoungZA** [comment]
@dorseypa1 @denisecoveyduc I am filling out the spreadsheet now and will notify you @dorseypa1 once complete.

**Comment by dorseypa1** [comment]
The new forms updates have been made in BigQuery.  The data will reflect in Domo in the next automatic processing cycle.

**Comment by denisecoveyduc** [comment]
Thank you all for your assistance! Can't wait to review the data as it rolls in.

**Comment by michelle-dooley** [comment]
@denisecoveyduc the data update came through as net new dashboard vs update to current.  We are resolving this issue now and will let you know once it is ready for review.  Apologize for the delays.  cc:  @dorseypa1

**Comment by denisecoveyduc** [comment]
Hey @michelle-dooley any updates on adding to current dashboard? or is there a link to the net new dashboard that I could review in the meantime? Thanks!

**Comment by michelle-dooley** [comment]
Hey @denisecoveyduc !!! Sorry I was out last week.  Anyway it is there :)  You should filter on the same form name "Form 5655 - Financial Status Report".  Note you will also see "Form 5655" in drop down, please ignore, we will remove so no confusion going forward. We are working on the backfill for history so it will go back further like other cards when that is complete.  But please take a look and let me know if you have any questions/concerns.

**Comment by HeatherRienks** [comment]
FYI - @Tom-Davis
